labgrid/driver/usbvideodriver.py

```diff
@@ -38,23 +44,36 @@ def select_caps(self, hint=None):
         raise InvalidConfigError("Unkown video format {} for device {:04x}:{:04x}".format(
             variant, self.video.vendor_id, self.video.model_id))
 
-    def get_pipeline(self):
+    def get_pipeline(self, path, caps, controls=None):
         match = (self.video.vendor_id, self.video.model_id)
         if match == (0x046d, 0x082d):
-            return "v4l2src device={} ! {} ! h264parse ! fdsink"
-        if match == (0x046d, 0x0892):
-            return "v4l2src device={} ! {} ! decodebin ! vaapipostproc ! vaapih264enc ! h264parse ! fdsink"  # pylint: disable=line-too-long
-        raise InvalidConfigError("Unkown USB video device {:04x}:{:04x}".format(*match))
+            controls = controls or "focus_auto=1"
+            inner = "h264parse"
+        elif match == (0x046d, 0x0892):
+            inner = "decodebin ! vaapipostproc ! vaapih264enc ! h264parse"
```

Comment:
This breaks this pipeline, however even on master this is currently displaying in green, do i'd be in favor of moving this to jpeg too and reworking the vaapi support eventually.

Reply:
@Emantor has switched this to jpeg.